### ISSUE TITLE
URL in <link> element

### ISSUE THREAD

**Issue by AlexErk** [ebu/radiodns-manager#133]
Ind SI.xml in the <serviceProvider> section the service providers web site is added as "url" attribute. 

cvc-complex-type.4: Attribute 'uri' must appear on element 'link'.

**Comment by magicbadger** [comment]
The attribute name inside the `link` element (under both `serviceProvider` and `service`) is misnamed.

This should be
```
<link uri="https://www.br.de" mimeValue="text/html" description="BR Webpage" xml:lang="de"/>
```

Rather than
```
<link url="https://www.br.de" mimeValue="text/html" description="BR Webpage" xml:lang="de"/>
```

**Comment by magicbadger** [comment]
closed in [release 4.1.1](https://github.com/ebu/radiodns-manager/releases/tag/4.1.1)
